### ISSUE TITLE
The debugger is not ready After update to 5.7.1

### ISSUE THREAD

**Issue by gholami-mohammad** [joefitzgerald/go-plus#726]
1. **Go plus can not connect to go-debug after updating to version 5.7.1**
here is console error
```
go-debug Failed to get "dlv" TypeError: Cannot create property 'timeout' on string 'project'
    at Executor.ensureOptions (/Users/mgh/.atom/packages/go-plus/executor.js:168:7)
    at Promise (/Users/mgh/.atom/packages/go-plus/executor.js:67:29)
    at Executor.exec (/Users/mgh/.atom/packages/go-plus/executor.js:66:12)
    at getVersion (/Users/mgh/.atom/packages/go-debug/lib/delve-get.js:31:30)
    at located (/Users/mgh/.atom/packages/go-debug/lib/delve-get.js:23:14)

```
1. **Autocomplete not work correctly.** In previous version if file has some error, autocomplete worked properly. But in this update it is not working correctly! :(

**Comment by zmb3** [comment]
I submitted a PR to go-debug to fix the first issue.

cc @lloiser for the autocomplete question

**Comment by gholami-mohammad** [comment]
Connecting to go-debug bug fixed after go-debug updated. 
**Still autocomplete not working correctly.**

**Comment by lloiser** [comment]
@gholami-mohammad can you provide the code and some steps to reproduce this?

Pls check if the console in the developer tools of atom contains anything suspicious. Open it with the command `Toggle Dev Tools` (`alt-cmd-i`)

**Comment by gholami-mohammad** [comment]
@lloiser  The is nothing suspicious . Just some info log like this:
```
Error: # ems_ms/controllers
./sync_common_tables_controller.go:395:38: "ems_ms/models".MoRegionAmbulanceTechnician literal.ID evaluated but not used
    at that.goconfig.locator.findTool.then.then.results (/Users/mgh/.atom/packages/go-plus/builder.js:101:22)
    at process._tickCallback (internal/process/next_tick.js:103:7)
/Users/mgh/.atom/packages/go-plus/lib/orchestrator.js:87 Error: # ems_ms/controllers
./sync_common_tables_controller.go:395:38: "ems_ms/models".MoRegionAmbulanceTechnician literal.ID evaluated but not used
    at that.goconfig.locator.findTool.then.then.results (/Users/mgh/.atom/packages/go-plus/builder.js:101:22)

```

**Comment by gholami-mohammad** [comment]
Code Autocomplete and import autocomplete bug fixed after update to go-plus v 5.7.2
